docs/learn/prompts.md

```diff
@@ -137,7 +137,7 @@ While Mirascope provides a consistent interface, support varies among providers:
 |---------------|-----------|--------|--------|------|---------|--------|
 | text          | ✓         | ✓      | ✓      | ✓    | ✓       | ✓      |
 | image         | ✓         | -      | ✓      | ✓    | ✓       | ✓      |
-| audio         | -         | -      | ✓      | -    | -       | -      |
+| audio         | -         | -      | ✓      | -    | -       | ✓      |
```

Comment:
i think we should add a little blurb here that the shorthand can handle both `pydub.AudioSegment` and `wave.Wave_read`? since we only show the `pydub.AudioSegment` example I worry users won't know they can use wave directly

either that or we should add tabs for wave vs. pydub as i mentioned before so they can see both examples